employees/common/exports.py

```diff
@@ -294,21 +290,10 @@ def save_work_book_as_csv(writer: _writer, work_book: Workbook, hours_column_set
         writer.writerow(next_row)
 
 
-def save_work_book_as_zip_of_csv(work_book: Workbook) -> BytesIO:
-    zip_file_data = BytesIO()
-    zip_file = zipfile.ZipFile(zip_file_data, "w")
-
-    for sheet_index, sheet_name in enumerate(work_book.sheetnames):
+def save_work_book_for_project_as_csv(work_book: Workbook, writer: _writer) -> None:
+    hours_column_setting: ColumnSettings = constants.HEADERS_TO_COLUMNS_SETTINGS_FOR_USER_IN_PROJECT.value[
```

Comment:
I suggest either make it a function with some very descriptive name or adding a comment above.